### ISSUE TITLE
Not able to make a turn around

### ISSUE THREAD

**Issue by rzi2001** [flybywiresim/aircraft#9582]
### Aircraft Version

Stable

### Build info

```json
{
    "built": "2024-11-21T09:20:02+00:00",
    "ref": "refs/heads/master",
    "sha": "1d42acddaf69e496f74e1ca1d6967183eaecf65c",
    "actor": "tracernz",
    "event_name": "manual",
    "pretty_release_name": "master:1d42acdd",
    "version": "a380x-v0.13.0-dev.1d42acd"
}
```

### Describe the bug

If you do a turn around, the Departure field is deactivated on the init page because the FOB field is missing on the Field & Load page. This may be related to the landing.
After landing I got the message: Fuel no ZFW or ZFWG...

### Expected behavior

Not able to make a turn around

### Steps to reproduce

Make a flight and try to make the next, without laoding the aircraft again

### References (optional)

![Screenshot 2024-11-23 145220](https://github.com/user-attachments/assets/6924b3ca-3c33-4908-9236-19134e1a13c7)
![Microsoft Flight Simulator 23 11 2024 18_35_43_1](https://github.com/user-attachments/assets/d42b0725-1818-4c3d-b673-e77c547ed1bb)
![Microsoft Flight Simulator 24 11 2024 12_50_49](https://github.com/user-attachments/assets/208eb3da-dfc2-4b96-87c5-3c2ae0804959)
![Microsoft Flight Simulator 24 11 2024 12_50_44](https://github.com/user-attachments/assets/43084504-9034-4555-96c1-c717e8cb1001)
![Microsoft Flight Simulator 24 11 2024 12_50_29](https://github.com/user-attachments/assets/a3b79c0e-4f27-4f95-9bb6-f0f7c6b3e3a9)

### Additional info (optional)

![Microsoft Flight Simulator 23 11 2024 18_35_43](https://github.com/user-attachments/assets/c764e54a-6372-4a02-8366-1de45b0ea570)
I've added 3 screenshots immediately after landing.

### Discord Username (optional)

rzi2001[Ralf]

**Comment by tracernz** [comment]
Can you check what data is left in the FMS after completing the first flight, before any new data is entered?

**Comment by rzi2001** [comment]
Yes it is not the stable Version, because i changed to the developerversion. But it is the same Problem  in the stable version, that i used before.

**Comment by rzi2001** [comment]
I've added 3 screenshots immediately after landing.

 

Von: Michael Corcoran ***@***.***> 
Gesendet: Sonntag, 24. November 2024 02:50
An: flybywiresim/aircraft ***@***.***>
Cc: rzi2001 ***@***.***>; Author ***@***.***>
Betreff: Re: [flybywiresim/aircraft] Not able to make a turn around (Issue #9582)

 

Can you check what data is left in the FMS after completing the first flight, before any new data is entered?

**Comment by rzi2001** [comment]
I had accidentally closed it

**Comment by Amorisia** [comment]
Hello, I tried to perform a turn around today. After landing, deboarding and reset the aircraft with navdata swap, I imported the new flight in the MCD DU INIt page without any problem. I was able to create the flight plan with departure and arrival as it was reset normally.

Everything was reset except the FUEL&LOAD page where I was unable to set the block fuel and the PERF page was stuck on GA phase. Then I was unable to enter any value in TO phase as everything was greyed out. After my second landing, I tried to reset the aircraft again with navdata swap but also switched off completely the aircraft about 15 minutes. Same status after switching on again.

**Comment by AngusIrwin** [comment]
Tried turning around for another flight. The Fuel & payload page hadn't reset so Block fuel cannot be entered causing the previously described cautions. The perf page also fails to reset, not allowing the entering of data for V-speeds or cruise speed. This is an issue on the stable version.

**Comment by Amorisia** [comment]
Was working perfectly on January 19/20 as I made full QFA1 flight with turnaround at WSSS.

**Comment by AngusIrwin** [comment]
I've started another flight and will test again, might not be consistent, or maybe is triggered by something.
